From 44b075199346e3357df86d68e3cae86406b95d94 Mon Sep 17 00:00:00 2001
From: Wasim Nazir <quic_wasimn@quicinc.com>
Date: Tue, 5 Nov 2024 19:39:01 +0530
Subject: [PATCH 1/3] QCLINUX: arm64: dts: qcom: Add msm-id & board-id for
 qcs9075-rb8

Enable DT identification support for qcs9075-rb8 board.

Signed-off-by: Wasim Nazir <quic_wasimn@quicinc.com>
Upstream-Status: Pending
---
 arch/arm64/boot/dts/qcom/qcs9075-rb8.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/qcs9075-rb8.dts b/arch/arm64/boot/dts/qcom/qcs9075-rb8.dts
index cac4f7252223..f9feb3e042be 100644
--- a/arch/arm64/boot/dts/qcom/qcs9075-rb8.dts
+++ b/arch/arm64/boot/dts/qcom/qcs9075-rb8.dts
@@ -14,6 +14,8 @@
 / {
 	model = "Qualcomm Technologies, Inc. Robotics RB8";
 	compatible = "qcom,qcs9075-rb8", "qcom,qcs9075", "qcom,sa8775p";
+	qcom,msm-id = <676 0x20000>;
+	qcom,board-id = <0x10020 0>, <0x10019 0>;
 
 	aliases {
 		serial0 = &uart10;
-- 
2.48.1

